app/components/loading/loading.modal.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
```

Comment:
the file name is with dot like all others file name in the project, so it is consistent.